releasing/releaser/vendor/kubeflow/automation/release.libsonnet

```diff
@@ -108,6 +108,14 @@
                 },
               },
             },
+            {
```

Comment:
can you make the same change in
kubeflow/automation/release.libsonnet

Otherwise your changes will be overwritten if we ever upgrade the release prototype

Reply:
Okay :)

Reply:
Thanks for the review